compiler/src/dune

```diff
@@ -4,3 +4,21 @@
 (menhir
  (flags "--table" "--explain")
  (modules parser))
+
+(library
+  (name jasmin_support)
+  (wrapped false)
```

Comment:
Question: what global names will these modules have? Something like `Jasmin.Location` for example, or just `Location`? Having `wrapped` set to true is usually recommended to avoid name clashes at linking time.